pkg/qiniu/mock.go

```diff
@@ -0,0 +1,77 @@
+/*
+ Copyright 2020 Qiniu Cloud (qiniu.com)
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package qiniu
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+
+	"github.com/julienschmidt/httprouter"
+	"github.com/qiniu/api.v7/v7/storage"
+	"github.com/sirupsen/logrus"
+)
+
+func MockQiniuServer(config *Config) (client *Client, router *httprouter.Router, serverURL string, teardown func()) {
```

Comment:
`mock.go` seems to be the utility for the tests, so i think it's better to move this file into `qnPresubmit_test.go`, since we don't need consider it's coverage.

Reply:
if move to qnPresubmit_test.go, MockQiniuServer() cannot be called by other package. In my situation, I also need call MockQiniuServer() in job_test.go